### DISCUSSION TITLE
Triggered an air raid, whenever it seems like it's about to end it just loops.

### DISCUSSION THREAD

**Discussion by RedburnDrift** [TEHE-Studios/ExpandedHelicopterEvents#142] [Q&A]
Is there a way to force stop an event? If not that sounds like it could be a cool feature. Would like a way to end this stupid looping air siren without restarting my server of 20+ people online.

**Comment by Chuckleberry-Finn** [comment]
Issues should be written within the issues section. That being said this is also a duplicate issue.

#85
